resources/function_help/json/set_color_part

```diff
@@ -8,5 +8,5 @@
 	{"arg":"component", "description":"a string corresponding to the color component to set. Valid options are:<br /><ul><li>red: RGB red component (0-255)</li><li>green: RGB green component (0-255)</li><li>blue: RGB blue component (0-255)</li><li>alpha: alpha (transparency) value (0-255)</li><li>hue: HSV hue (0-360)</li><li>saturation: HSV saturation (0-100)</li><li>value: HSV value (0-100)</li><li>hsl_hue: HSL hue (0-360)</li><li>hsl_saturation: HSL saturation (0-100)</li><li>lightness: HSL lightness (0-100)</li><li>cyan: CMYK cyan component (0-100)</li><li>magenta: CMYK magenta component (0-100)</li><li>yellow: CMYK yellow component (0-100)</li> <li>black: CMYK black component (0-100)</li></ul>"},
 	{"arg":"value", "description":"new value for color component, respecting the ranges listed above"}
 	          ],
-  "examples": [ { "expression":"set_color_part('200,10,30','green',50)", "returns":"200,50,30"}]
+  "examples": [ { "expression":"set_color_part('200,10,30','green',50)", "returns":"'200,50,30'"}]
```

Comment:
tested on 3.10.7 with Catalina, this actually returns
```suggestion
  "examples": [ { "expression":"set_color_part('200,10,30','green',50)", "returns":"'200,50,30,255'"}]
```

Reply:
I'd update the example to match